src/components/BookingComponent.jsx

```diff
@@ -1,57 +1,176 @@
 import React, { useEffect, useState } from "react";
+import moment from "moment";
+import { bookingType, formatAsZulu, getPositionFromFrequency } from "../utils/BookingHelper";
+import { ExternalLinkIcon } from './icons/ExternalLinkIcon';
 
 
 const BookingComponent = () => {
     const [ControlCenterBookings, setControlCenterBookings] = useState([]);
-    const [ControlCenterPositions, setControlCenterPositions] = useState([]);
     const [VatsimNetworkSessions, setVatsimNetworkSessions] = useState([]);
+    const [dateArray, setDateArray] = useState([]);
     const [loading, setLoading] = useState(true);
 
+    const acceptedFIRsRegex = /((EK[A-Z][A-Z]_)|(EF[A-Z][A-Z]_)|(BI[A-Z][A-Z]_)|(EN[A-Z][A-Z]_)|(ES[A-Z][A-Z]_))\w+/i;
```

Comment:
```suggestion
    const acceptedFIRsRegex = /((?:BI|EF|EK|EN|ES)([A-Z][A-Z]_))\w+/i;
```
suggestion(non-blocking): simplify the regex